handlers/main.yml

```diff
@@ -10,8 +10,13 @@
 - name: restart daemons
   command: /bin/true
   notify:
+    - reset failed kubelet
     - restart kubelet
 
+- name: reset failed kubelet
+  become: true
+  command: systemctl reset-failed kubelet
```

Comment:
may be notify `restart kubelet` here so in the tasks you only need to call `reset failed kubelet`

Reply:
Fixed as requested.